### PR TITLE
Update app service and ui packages

### PR DESCRIPTION
**Pull request by EricJizbaMSFT**
Breaking change in app service was just a change to the signature of "deploy".

Also switch away from deprecated tslint extension. See here for more info: https://github.com/Microsoft/vscode-typescript-tslint-plugin#differences-with-the-vscode-tslint-extension

### CODE REVIEW

File: .vscode/extensions.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
     "recommendations": [
-        "eg2.tslint",
+        "ms-vscode.vscode-typescript-tslint-plugin",
```
Comment (PrashanthCorp) [inline .vscode/extensions.json]: Is this related to the other changes in this PR?

Comment (EricJizbaMSFT) [reply .vscode/extensions.json]: ¯\\\_(ツ)\_/¯ It's all related in the sense that it's tedious process-related changes
